client_it_test.go

```diff
@@ -321,3 +321,112 @@ func TestClusterReconnection_ReconnectModeOff(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 	assert.Equal(t, false, c.Running())
 }
+
+func TestClient_GetDistributedObjects(t *testing.T) {
+	const (
+		testMapName = "get-distributed-objects-map"
+		testSetName = "get-distributed-objects-set"
+	)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 40*time.Second)
```

Comment:
Is there a reason a  context with 40s timeout is created? How about using `context.Background()`?

Reply:
I assumed context timeout is used as the test timeout value based on the other tests through the file. And since multiple clients are created within the tests, I set it to 40s. I can change it to background if it's not supposed to be that way - I mean the test timeout duration.